server/app/controllers/api/v1/auth_controller.rb

```diff
@@ -6,9 +6,9 @@ def register
     )
 
     if result.success?
-      render :json => { errors: [], resource: result.user }, :status => :created
+      render json: { errors: [], resource: result.user }, status: :created
```

Comment:
Are we okay with not using hash rockets? I still have a tendency to use them because that's what I'm used to but this helps unify them and autocorrects them.